reader/views.py

```diff
@@ -53,7 +53,6 @@ def hit_count(request):
     return HttpResponse(json.dumps({}), content_type="application/json")
 
 
-@cache_control(public=True, max_age=60, s_maxage=60)
```

Comment:
I think this is not meant to be there, it crashes when I tried to run it locally.